config/src/config/consensus_config.rs

```diff
@@ -89,6 +89,8 @@ pub struct ConsensusConfig {
     pub rand_rb_config: ReliableBroadcastConfig,
     pub num_bounded_executor_tasks: u64,
     pub enable_pre_commit: bool,
+
+    pub max_pending_commit_votes_in_cache: u64,
```

Comment:
The name is confusing. It should be the rounds but right now it means the number of votes?

Reply:
Done.